src/Admin.php

```diff
@@ -25,4 +25,89 @@ public static function admin_notices() {
     <?php }
   }
 
+  /**
+   * Returns products backlinking to a given accessory product.
+   */
+  public static function wp_ajax_related_accessories_backlinks() {
+    global $wpdb;
+    $postId = intval($_POST['post']);
+    $html = '<ol>';
```

Comment:
Invalid HTML / output in case of no posts. Please move this into the have_posts() condition.